pkg/config/config.go

```diff
@@ -619,6 +622,16 @@ type Destination struct {
 	IsMachine bool `toml:"is_machine,omitempty"`
 }
 
+// Consumes container image's os and arch and returns list of configured runtimes which
+// can be used for following container image
+func (c *EngineConfig) ImagePlatformToRuntimes(os string, arch string) string {
+	platformString := os + "/" + arch
+	if val, ok := c.PlatformToOCIRuntime[platformString]; ok {
+		return val
+	}
+	return ""
```

Comment:
Why isn't this returning a list of runtimes to be used with the container image in the default case?  You are making the caller do extra work that  common should do.
Caller is going to call ociRuntime=cfg.ImagePlatformToRuntime(os, arch)
They should then have the runtime, no special handling.

Reply:
Default is already set when the runtime is created at podman end, this only returns runtime if any `platform` string had any dedicated runtime configured otherwise nothing. At `podman` end we only set dedicated runtime if found otherwise we do nothing.

See: https://github.com/containers/podman/pull/16434/files#diff-3ff238a49e99bd584fa865688b0fd9914acd5cdd71f46606c6750a855da6cf4fR128

Reply:
I can return `c.OCIRuntime` I guess that should work.